<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>579</width>
    <height>577</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#frmLogin,QWidget#frmPopup,QWidget#frmHostInfo,QWidget#frmLogout,QWidget#frmConfig,QWidget#frmData,QWidget#frmDefence,QWidget#frmHost,QWidget#frmMain,QWidget#frmPwd,QWidget#frmSelect,QWidget#frmMessageBox{
	border:1px solid #454648;
	border-radius:0px;	
}

.QFrame{
	border:1px solid #B8B8B8;
	border-radius:5px;
}

QWidget#widget_title{
	background: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 #454648, stop:1 #7A7A7A);
}

QLabel#lab_Ico,QLabel#lab_Title{
	border-radius:0px;
	color: #F0F0F0;
	background-color:rgba(0,0,0,0);
	border-style:none;
}

QLineEdit {
	border: 1px solid #B8B8B8;
	border-radius: 5px;
	padding: 2px;
	background: none;
	selection-background-color: #454648;
}

QLineEdit[echoMode=&quot;2&quot;] { 
	lineedit-password-character: 9679; 
}

.QGroupBox{
	border: 1px solid #B8B8B8;
	border-radius: 5px;
}

.QPushButton{
	border-style: none;
	border: 0px;
	color: #F0F0F0;
	padding: 5px;	
	min-height: 20px;
	border-radius:5px;
	background: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 #454648, stop:1 #7A7A7A); 
}

.QPushButton[focusPolicy=&quot;0&quot;] {
	border-style: none;
	border: 0px;
	color: #F0F0F0;
	padding: 0px;	
	min-height: 10px;
	border-radius:3px;
	background: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 #454648, stop:1 #7A7A7A); 
}

.QPushButton:hover{ 
	background: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 #B8B8B8, stop:1 #D6D6D6);
}

.QPushButton:pressed{ 
	background: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 #454648, stop:1 #7A7A7A);
}

QPushButton#btnMenu,QPushButton#btnMenu_Min,QPushButton#btnMenu_Max,QPushButton#btnMenu_Close{
	border-radius:0px;
	color: #F0F0F0;
	background-color:rgba(0,0,0,0);
	border-style:none;
}

QPushButton#btnMenu:hover,QPushButton#btnMenu_Min:hover,QPushButton#btnMenu_Max:hover{
	background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgba(25, 134, 199, 0), stop:1 #B8B8B8);
}

QPushButton#btnMenu_Close:hover{
	background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgba(238, 0, 0, 128), stop:1 rgba(238, 44, 44, 255));
}
QRadioButton {
	spacing: 2px;
}

QRadioButton::indicator {
	width: 15px; 
	height: 15px; 
}

QRadioButton::indicator::unchecked {
	image: url(:/qss_icons/img_rc/radio_normal.png); 
}

QRadioButton::indicator::checked {
	image: url(:/qss_icons/img_rc/radio_selected.png); 
}

QComboBox,QDateEdit{
	border-radius: 3px;
	padding: 1px 10px 1px 5px;
	border: 1px solid #B8B8B8;
}

QComboBox::drop-down,QDateEdit::drop-down {
	subcontrol-origin: padding;
	subcontrol-position: top right;
	width: 15px; 
	border-left-width: 1px;
	border-left-style: solid;
	border-top-right-radius: 3px;
	border-bottom-right-radius: 3px;
	border-left-color: #B8B8B8;
}

QComboBox::down-arrow,QDateEdit::down-arrow {
	image: url(:/qss_icons/img_rc/array_down.png); 
}

QMenu {
	background-color:#F0F0F0;
	margin: 2px;
}

QMenu::item {	
	padding: 2px 12px 2px 12px;
}

QMenu::indicator {
	width: 13px;
	height: 13px;
}

QMenu::item:selected {
	color: #F0F0F0;
	background: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 #454648, stop:1 #7A7A7A); 
}

QMenu::separator {
	height: 1px;
	background: #B8B8B8;
}

QProgressBar {
	border-radius: 5px;
	text-align: center;
	border: 1px solid #B8B8B8;
}

QProgressBar::chunk {
	width: 5px; 
	margin: 0.5px;
	background-color: #454648;
}

QSlider::groove:horizontal,QSlider::add-page:horizontal { 
	background: #808080; 
	height: 8px; 
	border-radius: 3px; 
}

QSlider::sub-page:horizontal {
	height: 8px; 
	border-radius: 3px;
	background: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 #454648, stop:1 #7A7A7A); 
}

QSlider::handle:horizontal {
	width: 13px; 
	margin-top: -3px; 
	margin-bottom: -3px; 
	border-radius: 6px;
	background: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:0.5, fx:0.5, fy:0.5,stop:0.6 #F0F0F0, stop:0.778409 #B8B8B8);
}

QSlider::handle:horizontal:hover {
	background: qradialgradient(spread: pad, cx: 0.5, cy: 0.5, radius: 0.5, fx: 0.5, fy: 0.5, stop: 0.6 #F0F0F0,stop:0.778409 #454648);
}

QSlider::groove:vertical,QSlider::sub-page:vertical {
	background:#808080; 
	width: 8px; 
	border-radius: 3px;
}

QSlider::add-page:vertical {
	width: 8px;
	border-radius: 3px;
	background: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 #454648, stop:1 #7A7A7A); 
}

QSlider::handle:vertical {
	height: 14px; 
	margin-left: -3px;
	margin-right: -3px;
	border-radius: 6px;
	background: qradialgradient(spread: pad, cx: 0.5, cy: 0.5, radius: 0.5, fx: 0.5, fy: 0.5, stop: 0.6 #F0F0F0, stop:0.778409 #B8B8B8);
}

QSlider::handle:vertical:hover {
	background: qradialgradient(spread: pad, cx: 0.5, cy: 0.5, radius: 0.5, fx: 0.5, fy: 0.5, stop: 0.6 #F0F0F0,stop:0.778409 #454648);
}

QScrollBar:vertical {
	width:10px; 
	background-color:rgba(0,0,0,0%); 
	padding-top:10px; 
	padding-bottom:10px;
}

QScrollBar:horizontal {
	height:10px; 
	background-color:rgba(0,0,0,0%); 
	padding-left:10px; padding-right:10px;
}

QScrollBar::handle:vertical {
	width:10px;
	background: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 #B8B8B8, stop:1 #D6D6D6); 
}

QScrollBar::handle:horizontal {
	height:10px;
	background: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 #B8B8B8, stop:1 #D6D6D6); 
}

QScrollBar::handle:vertical:hover {
	width:10px;
	background: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 #454648, stop:1 #7A7A7A); 
}

QScrollBar::handle:horizontal:hover {
	height:10px;
	background: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 #454648, stop:1 #7A7A7A); 
}

QScrollBar::add-line:vertical {
	height:10px;
	width:10px;
	subcontrol-position: bottom; 
	subcontrol-origin: margin;
	border-image:url(:/qss_icons/img_rc/add-line_vertical.png);
}

QScrollBar::add-line:horizontal {
	height:10px;
	width:10px;
	subcontrol-position: right;
	subcontrol-origin: margin;
	border-image:url(:/qss_icons/img_rc/add-line_horizontal.png);
}

QScrollBar::sub-line:vertical {
	height:10px;
	width:10px;
	subcontrol-position: top; 
	subcontrol-origin: margin;
	border-image:url(:/qss_icons/img_rc/sub-line_vertical.png);
}

QScrollBar::sub-line:horizontal {
	height:10px;
	width:10px;
	subcontrol-position: left;
	subcontrol-origin: margin;
	border-image:url(:/qss_icons/img_rc/sub-line_horizontal.png);
}

QScrollBar::add-page:vertical,QScrollBar::sub-page:vertical {
	width:10px;
	background: #C0C0C0;
}

QScrollBar::add-page:horizontal,QScrollBar::sub-page:horizontal {
	height:10px;
	background: #C0C0C0;
}

QScrollArea {
	border: 0px ; 
}

QTreeView,QListView,QTableView{
	border: 1px solid #B8B8B8; 
	selection-background-color: #454648;
	selection-color: #F0F0F0;
}

QTableView::item:selected, QListView::item:selected, QTreeView::item:selected {
	color: #F0F0F0;
	background: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 #454648, stop:1 #7A7A7A); 
}

QTableView::item:hover, QListView::item:hover, QTreeView::item:hover {
	color: #F0F0F0;
	background: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 #B8B8B8, stop:1 #D6D6D6); 
}

QTableView::item, QListView::item, QTreeView::item {
	padding: 5px; 
	margin: 0px; 
}

QHeaderView::section {
	padding:3px;
	margin:0px;
	color:#F0F0F0;
	border: 1px solid #F0F0F0;
	background: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 #B8B8B8, stop:1 #D6D6D6);
}

QTabBar::tab {
	border-bottom-left-radius:0px;
	border-bottom-right-radius:0px;
	color: #F0F0F0;
	min-width: 60px;
	min-height: 20px;
	padding: 3px 8px 3px 8px;
	margin:1px;
	background: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 #B8B8B8, stop:1 #D6D6D6); 
}

QTabBar::tab:selected, QTabBar::tab:hover {
	background: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 #454648, stop:1 #7A7A7A); 
}

QStatusBar::item {
     border: 1px solid #B8B8B8;
     border-radius: 3px;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QLabel" name="title">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>150</height>
      </size>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:28pt; font-weight:600;&quot;&gt;SIGN&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:28pt; font-weight:600;&quot;&gt;UP&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="leftMargin">
      <number>20</number>
     </property>
     <property name="rightMargin">
      <number>20</number>
     </property>
     <item>
      <widget class="QLabel" name="lbl_reply">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>30</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>MS PGothic</family>
         <pointsize>9</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="txt_user">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>40</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>poppins</family>
         <pointsize>-1</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">padding-left: 5px;
font-family: poppins;
font-size: 16px</string>
       </property>
       <property name="placeholderText">
        <string>Enter a Username</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="txt_email">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>40</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">padding-left: 5px;
font-family: poppins;
font-size: 16px</string>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Normal</enum>
       </property>
       <property name="placeholderText">
        <string>Enter e-mail</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="txt_fname">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>40</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">padding-left: 5px;
font-family: poppins;
font-size: 16px</string>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Normal</enum>
       </property>
       <property name="placeholderText">
        <string>Enter fullname</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="txt_pass">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>40</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">padding-left: 5px;
font-family: poppins;
font-size: 16px</string>
       </property>
       <property name="inputMethodHints">
        <set>Qt::ImhHiddenText|Qt::ImhNoAutoUppercase|Qt::ImhNoPredictiveText|Qt::ImhSensitiveData</set>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
       <property name="placeholderText">
        <string>Enter a password</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="txt_repass">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>40</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">padding-left: 5px;
font-family: poppins;
font-size: 16px</string>
       </property>
       <property name="inputMethodHints">
        <set>Qt::ImhHiddenText|Qt::ImhNoAutoUppercase|Qt::ImhNoPredictiveText|Qt::ImhSensitiveData</set>
       </property>
       <property name="placeholderText">
        <string>Re-enter a password</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="reg_button">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>10</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Poppins</family>
         <pointsize>8</pointsize>
         <weight>50</weight>
         <bold>false</bold>
         <kerning>true</kerning>
        </font>
       </property>
       <property name="text">
        <string>Sign Up</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnlogin">
       <property name="styleSheet">
        <string notr="true">background:transparent;
color:dodgerblue;
font-weight: 500;
text-decoration:underline;
font-size:12px;
</string>
       </property>
       <property name="text">
        <string>Already Have an account. Sign in.</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
